gpt/gpt/gpt.py

```diff
@@ -66,6 +72,28 @@ def signup(self):
         return pc.redirect("/home")
 
     def get_result(self):
+        if (
+            pc.session()
+            .query(Question)
+            .where(Question.username == self.username)
+            .where(Question.prompt == self.prompt)
+            .first()
+            or len(
+                pc.session()
+                .query(Question)
+                .where(Question.username == self.username)
+                .where(
+                    Question.timestamp
+                    > datetime.datetime.now() - datetime.timedelta(days=1)
+                )
+                .all()
```

Comment:
You can do `.count()` here instead to do the operation within sql, and remove the `len(...)`